src/hotspot/cpu/x86/x86.ad

```diff
@@ -1610,13 +1610,9 @@ bool Matcher::match_rule_supported(int opcode) {
       break;
     case Op_CopySignD:
     case Op_CopySignF:
-      if (UseAVX < 3 || !is_LP64)  {
+      if (UseAVX < 1 || !is_LP64)  {
```

Comment:
Should it be limited to just AVX2, or can the new rules work on AVX1 as well? Since they only use instructions that are available to AVX1.